WordPressKit/WordPressComRestApi.swift

```diff
@@ -24,10 +26,19 @@ import Alamofire
     case preconditionFailure
 }
 
-open class WordPressComRestApi: NSObject {    
-    @objc public static let ErrorKeyErrorCode: String = "WordPressComRestApiErrorCodeKey"
-    @objc public static let ErrorKeyErrorMessage: String = "WordPressComRestApiErrorMessageKey"
-    @objc public static let SessionTaskKey: String = "WordPressComRestAPI.sessionTask"
+// MARK: - WordPressComRestApi
+
+open class WordPressComRestApi: NSObject {
+
+    // MARK: Properties
+
+    @objc public static let ErrorKeyErrorCode       = "WordPressComRestApiErrorCodeKey"
+    @objc public static let ErrorKeyErrorMessage    = "WordPressComRestApiErrorMessageKey"
+
+    @objc public static let LocaleKeyDefault        = "locale"
+    @objc public static let LocaleKeyV2             = "_locale"
```

Comment:
Maybe just a one-line comment to clarify that version 2 endpoints use the `_locale` key instead of `locale`?